src/bindings/config.ts

```diff
@@ -51,6 +54,10 @@ export const loadConfig = async (context: Context): Promise<BotConfig> => {
       autoPayMode: process.env.AUTO_PAY_MODE === "TRUE" ? true : configFile?.autoPayMode ?? true,
       analyticsMode: process.env.ANALYTICS_MODE === "TRUE" ? true : configFile?.analyticsMode ?? false,
     },
+    sodium: {
+      publicKey: process.env.X25519_PUBLIC_KEY ?? "",
+      privateKey: process.env.X25519_PRIVATE_KEY ?? "",
+    },
```

Comment:
I am not fully understanding why we setup both keys. Both of them wouldn't be put in the partner repo.

Reply:
We're using `libsodium`, hence we need both the private key and public key,
follow the linked PR to generate one.

It's to be noted that the partners don't need to generate those keys, and those keys are only meant to 
be for bot constants (for instance: the _**installation key**_ is also a bot constant)

Reply:
Public keys should be able to be derived from private keys. We shouldn't burden the users to pass in more information than absolutely needed to operate the bot.

Reply:
hmm I think I understood. so you were trying to use asymmetric encryption method. that's good.

Reply:
> Public keys should be able to be derived from private keys. We shouldn't burden the users to pass in more information than absolutely needed to operate the bot.

the end users don't need to pass those values at all